metrics/web/lib/dashboard/presentation/widgets/build_result_bar.dart

```diff
@@ -54,23 +54,26 @@ class _BuildResultBarState extends State<BuildResultBar> {
 
   /// A [GraphIndicator] widget to use based on the [BuildResultBar.buildResult].
   Widget get _graphIndicator {
+    if (widget.buildResult == null || widget.buildResult.buildStatus == null) {
+      return null;
+    }
+
     switch (widget.buildResult.buildStatus) {
       case BuildStatus.successful:
         return const PositiveGraphIndicator();
-      case BuildStatus.cancelled:
-        return const NeutralGraphIndicator();
       case BuildStatus.failed:
         return const NegativeGraphIndicator();
+      case BuildStatus.unknown:
       default:
```

Comment:
It seems like we decided to get rid of the default case. The main reason is that if there is a new `BuildStatus` value, the Dart analyzer doesn't warn of a missing case if there is the `default` case in a `switch`.